views/layouts/authentication.tpl

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+
+<html class="layout-authentication">
+<head>
+  <title>Google Scraper</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+  {{ assets_css "/css/application.css" }}
+</head>
+<body class="{{.ControllerName}} {{.ActionName}}">
+  <div class="icon-sprite" hidden="true">
+    {{ render_file "static/symbol/svg/sprite.symbol.svg" }}
+  </div>
+
+  <div class="app-container">
```

Comment:
Avoid wrapping divs at all. This is an old technique that is not a best practice anymore since it cannot cater to all designs. Instead, prefer stacking each layout component and then add a container inside:

| To Avoid | Better |
|-|-|
| ![Screen Shot 2564-02-08 at 14 03 35](https://user-images.githubusercontent.com/696529/107186358-b09f8080-6a16-11eb-8d34-a248ae26dc6f.png) | ![Screen Shot 2564-02-08 at 14 02 05](https://user-images.githubusercontent.com/696529/107186377-bbf2ac00-6a16-11eb-939e-a0ab93224b49.png) |

Reply:
@olivierobert Fixed by removing `app-container` and moved paddings to `app-header` and `app-content` ` on [44b4d83](https://github.com/carryall/go-google-scraper-challenge/pull/25/commits/44b4d835a1cf95e6adf5078d251d42d628d72803)